### PR TITLE
Add temperature attribute to ATAOS applyCorrection

### PR DESCRIPTION
**Pull request by gmegh**

### CODE REVIEW

**Review by tribeiro — CHANGES_REQUESTED** [review]
Please, add temperature attributes to all the relevant events e.g.: 

- https://github.com/lsst-ts/ts_xml/blob/a16574f2a257c25557144337a94d9ba50605b017/python/lsst/ts/xml/data/sal_interfaces/ATAOS/ATAOS_Events.xml#L66
- https://github.com/lsst-ts/ts_xml/blob/a16574f2a257c25557144337a94d9ba50605b017/python/lsst/ts/xml/data/sal_interfaces/ATAOS/ATAOS_Events.xml#L92
- https://github.com/lsst-ts/ts_xml/blob/a16574f2a257c25557144337a94d9ba50605b017/python/lsst/ts/xml/data/sal_interfaces/ATAOS/ATAOS_Events.xml#L118
- https://github.com/lsst-ts/ts_xml/blob/a16574f2a257c25557144337a94d9ba50605b017/python/lsst/ts/xml/data/sal_interfaces/ATAOS/ATAOS_Events.xml#L144
- https://github.com/lsst-ts/ts_xml/blob/a16574f2a257c25557144337a94d9ba50605b017/python/lsst/ts/xml/data/sal_interfaces/ATAOS/ATAOS_Events.xml#L170
- https://github.com/lsst-ts/ts_xml/blob/a16574f2a257c25557144337a94d9ba50605b017/python/lsst/ts/xml/data/sal_interfaces/ATAOS/ATAOS_Events.xml#L231
- https://github.com/lsst-ts/ts_xml/blob/a16574f2a257c25557144337a94d9ba50605b017/python/lsst/ts/xml/data/sal_interfaces/ATAOS/ATAOS_Events.xml#L292
- https://github.com/lsst-ts/ts_xml/blob/a16574f2a257c25557144337a94d9ba50605b017/python/lsst/ts/xml/data/sal_interfaces/ATAOS/ATAOS_Events.xml#L318

**Comment by gmegh** [comment]
I think I have added them to all of the events

File: doc/news/interface_changes/DM-46190.ataos.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Add temperature attribute to ATAOS_command_applyCorrection.
```
Comment (tribeiro) [inline doc/news/interface_changes/DM-46190.ataos.rst:1]: Please, mention that you are also adding it to the events.

**Review by tribeiro — APPROVED** [review]
looks good

**Pull request merged**